sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Profile.scala

```diff
@@ -154,7 +154,7 @@ object Profile {
       def fromProfile(profile: Profile) = {
         import scala.collection.mutable.HashMap
 
-        val frames = ArrayList.empty[FrameJson]
```

Comment:
Do these changes introduce a benefit? Asking as its not immediately obvious to myself.